### DISCUSSION TITLE
[time-series chart]area chart style variation

### DISCUSSION THREAD

**Discussion by junlincc** [apache/superset#18419] [Ideas]
the new Echarts time-series chart looks great, but we should consider hardening it and increase time-series area chart style variation(Stack, Stream and Expand) to reach NVD3 area chart feature parity
after that, we can deprecate the old chart that has multiple issues not worth fixing
related https://github.com/apache-superset/superset-ui/issues/799

**Echarts - timeseries area chart**:
<img width="1791" alt="Screen Shot 2020-11-17 at 6 33 31 PM" src="https://user-images.githubusercontent.com/67837651/99475511-8baddc80-2903-11eb-99d3-afc97f4ef88a.png">

**NVD3 area chart**:
<img width="1471" alt="Screen Shot 2020-11-17 at 6 26 51 PM" src="https://user-images.githubusercontent.com/67837651/99475367-412c6000-2903-11eb-87f8-41643dd460c5.png">
<img width="1469" alt="Screen Shot 2020-11-17 at 6 26 43 PM" src="https://user-images.githubusercontent.com/67837651/99475372-42f62380-2903-11eb-841b-7bc2ec842cee.png">
<img width="1469" alt="Screen Shot 2020-11-17 at 6 26 58 PM" src="https://user-images.githubusercontent.com/67837651/99475380-44bfe700-2903-11eb-9e86-5f6ba58866c2.png">

**Comment by junlincc** [comment]
@willbarrett @nytai this can be good first Echarts tasks, what do you guys think? 
https://echarts.apache.org/examples/zh/editor.html?c=themeRiver-basic
https://echarts.apache.org/examples/zh/editor.html?c=area-stack

cc: @rusackas

**Comment by willbarrett** [comment]
Let's see what shakes out after we get ramped up. This looks like a good candidate for the hardening phase in January to me, but we'll see what the rest of the team thinks!
